Atomic/install.py

```diff
@@ -40,6 +43,10 @@ def cli(subparser):
         "%s'" % Install.print_install())
     installp.set_defaults(_class=Install, func='install')
     add_opt(installp)
+    installp.add_argument("--storage", dest="storage", default=None,
+                   help=_("Specify the storage. Default is currently '%s'.  You can"
+                          " change the default by editing /etc/atomic.conf and changing"
+                          " the 'default_storage' field." % _storage))
```

Comment:
I'd prefer not to right now.  We don't have a great way to do it.  I can add later if you like.